### PR TITLE
[pipeline] Add confirmation to pipeline scripts

### PR DESCRIPTION
**Pull request by Daniel-VDM**
* Add confirmation to restart a shard
* Remove profile from env vars for running on shard
and add confirmation for cmd ran on shard

```
[ec2-user@ip-172-31-37-52 pipeline]$ ./run_on_shard.sh -p os -T 0 'tac ../tmp_log/*/zerolog*.log | grep -m 1 HOORAY'
profile: os
execute: 'tac ../tmp_log/*/zerolog*.log | grep -m 1 HOORAY'
[Y]/n > Y
run_on_shard.sh: using timestamp 20200228.034942 (from logs/os -> /home/ec2-user/experiment-deploy/pipeline/logs/20200228.034942)
run_on_shard.sh: using outdir logs/os/run_on_shard/2020-03-04T04:59:55Z.AwhpZg
{"level":"info","port":"9000","ip":"34.228.217.167","myEpoch":42,"myBlock":3173,"myViewID":3173,"phase":0,"mode":"Normal","blockNum":3172,"epochNum":42,"ViewId":3172,"blockHash":"0xcd070746fc944634c63b00b770f57f350ccdb1dae7f6dbe3fa97f348b0d3af8b","index":0,"numTxns":0,"numStakingTxns":0,"caller":"/home/ec2-user/harmony/consensus/consensus_v2.go:188","time":"2020-03-04T04:59:49.482214344Z","message":"HOORAY!!!!!!! CONSENSUS REACHED!!!!!!!"}
18.234.25.135 returned status 1
52.90.35.200 returned status 1
18.212.20.105 returned status 1
3.88.43.69 returned status 1
3.85.39.99 returned status 1
3.91.175.210 returned status 1
3.85.131.224 returned status 1
54.196.176.89 returned status 1
3.86.233.138 returned status 1
3.89.218.142 returned status 1
52.91.168.71 returned status 1
54.89.211.35 returned status 1
3.82.246.160 returned status 1
18.212.34.121 returned status 1
3.87.5.212 returned status 1
52.87.240.173 returned status 1
52.90.115.143 returned status 1
54.85.1.236 returned status 1
34.239.125.19 returned status 1
3.86.163.194 returned status 1
54.211.223.125 returned status 1
34.207.94.186 returned status 1
54.146.188.99 returned status 1
54.146.184.202 returned status 1
18.206.161.102 returned status 1
run_on_shard.sh: results are in logs/os/run_on_shard/2020-03-04T04:59:55Z.AwhpZg
[ec2-user@ip-172-31-37-52 pipeline]$ ./run_on_shard.sh -p os -y -T 0 'tac ../tmp_log/*/zerolog*.log | grep -m 1 HOORAY'
profile: os
execute: 'tac ../tmp_log/*/zerolog*.log | grep -m 1 HOORAY'
run_on_shard.sh: using timestamp 20200228.034942 (from logs/os -> /home/ec2-user/experiment-deploy/pipeline/logs/20200228.034942)
run_on_shard.sh: using outdir logs/os/run_on_shard/2020-03-04T05:00:04Z.8KLaul
{"level":"info","port":"9000","ip":"34.228.217.167","myEpoch":42,"myBlock":3175,"myViewID":3175,"phase":0,"mode":"Normal","blockNum":3174,"epochNum":42,"ViewId":3174,"blockHash":"0x54d053613565447825529de1303f406fb383fce2308af47abfe39582fb5045c4","index":0,"numTxns":0,"numStakingTxns":0,"caller":"/home/ec2-user/harmony/consensus/consensus_v2.go:188","time":"2020-03-04T05:00:05.479258987Z","message":"HOORAY!!!!!!! CONSENSUS REACHED!!!!!!!"}
18.234.25.135 returned status 1
52.90.35.200 returned status 1
18.212.20.105 returned status 1
3.88.43.69 returned status 1
3.85.39.99 returned status 1
3.91.175.210 returned status 1
3.85.131.224 returned status 1
54.196.176.89 returned status 1
3.86.233.138 returned status 1
3.89.218.142 returned status 1
52.91.168.71 returned status 1
54.89.211.35 returned status 1
3.82.246.160 returned status 1
18.212.34.121 returned status 1
3.87.5.212 returned status 1
52.87.240.173 returned status 1
52.90.115.143 returned status 1
54.85.1.236 returned status 1
34.239.125.19 returned status 1
3.86.163.194 returned status 1
54.211.223.125 returned status 1
34.207.94.186 returned status 1
54.146.188.99 returned status 1
54.146.184.202 returned status 1
18.206.161.102 returned status 1
run_on_shard.sh: results are in logs/os/run_on_shard/2020-03-04T05:00:04Z.8KLaul
```

### CODE REVIEW

File: pipeline/restart_node.sh [hunk 1]
```diff
@@ -52,18 +52,21 @@ print_usage() {
 		-P          disable public RPC
 		-M          support multi-key
                   (default: ${default_multi_key})
+		-M          support multi-key
```
Comment (LeoHChen) [inline pipeline/restart_node.sh]: dup.

File: pipeline/run_on_shard.sh [hunk 2]
```diff
@@ -11,13 +11,10 @@ esac
...
 . "${progdir}/msg.sh"
 . "${progdir}/usage.sh"
-. "${progdir}/common_opts.sh"
 
 print_usage() {
 	cat <<- ENDEND
-		usage: ${progname} ${common_usage} [-t timestamp] [-qTOESrM] shard command
-
-		${common_usage_desc}
+		usage: ${progname} [-t timestamp] [-qTOESrM] shard command
```
Comment (LeoHChen) [inline pipeline/run_on_shard.sh]: add -py in the usage line.

File: pipeline/run_on_shard.sh [hunk 3]
```diff
@@ -82,6 +83,17 @@ case $# in
 	;;
 esac
 
+logdir="logs/${net_profile}"
```
Comment (LeoHChen) [inline pipeline/run_on_shard.sh]: did you check if net_profile is empty? if it is not set, you shall exit.

File: pipeline/restart_node.sh [hunk 4]
```diff
@@ -116,6 +120,16 @@ shift 1
 
 log_define -v restart_node_log_level -l INFO rn
 
+echo "profile: ${HMY_PROFILE}"
```
Comment (LeoHChen) [inline pipeline/restart_node.sh]: do you force to pass '-p' option?  you'd force to pass in '-p' command option.

File: pipeline/restart_node.sh (hunk 1 above)
Comment (Daniel-VDM) [reply pipeline/restart_node.sh]: removed

File: pipeline/run_on_shard.sh (hunk 2 above)
Comment (Daniel-VDM) [reply pipeline/run_on_shard.sh]: added

File: pipeline/restart_node.sh (hunk 4 above)
Comment (Daniel-VDM) [reply pipeline/restart_node.sh]: checked for a profile so that one has to set it to run the rest of the script

**Pull request merged**